draft-rha-jose-hpke-encrypt.md

```diff
@@ -208,51 +219,47 @@ This example demonstrates the representaton of an encapsulated key as a JWK.
 
 ### Integrated Encryption
 
-In Integrated Encryption mode, HPKE is employed to directly encrypt the (compressed) plaintext, and the resulting ciphertext is included in the JWE ciphertext. 
+This mode only supports a single recipient and JWE Compact serialization.
```

Comment:
Why does a mode only support Compact? Any mode supporting compact should also support flattened?

Reply:
Agreed, we have an issue to track it here: https://github.com/tireddy2/JOSE_HPKE/issues/15

Reply:
Hmm, actually no agreed... unless you count empty string "iv", "tag" as being ok?

Reply:
https://github.com/tireddy2/JOSE_HPKE/pull/19/files#r1485646508